_posts/2021-06-22-choosing-right-transformation.md

```diff
@@ -0,0 +1,114 @@
+
+ArcMap is asking you to choose a transformation. ___By skipping this, no transformation is used at all. The underlying difference between the two datums is left for all to see:___
+
+![Default when no transformation is selected]({% link images/transformations_default.jpg %}){: .flex .flex--center}
+
+_(Yes, I realize this is in ArcGIS Pro, but the result is the same)_
+{: .flex .flex--center}
+
+ArcGIS Pro is better about this... and worse, actually. If you add Web Mercator data to a UTM 12N NAD83 map, it happily chooses a transformation for you. However, by default it chooses the `WGS 1984 (ITRF00) To NAD 1983` transformation. While very, very similar, this still produces the 0.03 ft difference seen in the screenshot at the top of the page.
+
+![ArcGIS Pro's default transformation]({% link images/transformations_prodefault.jpg %}){: .flex .flex--center}
+
+To be frank, at these distances, the error from using the default transformation is much less than the potential error in the original data. However, it is just large enough to be outside of the default geodatabase tolerances. If you're doing something like a dissolve or split between the two datasets, or trying to enforce topology, you'll get annoying, almost invisible slivers and gaps.
+
+## Can You At Least Give Me a Warning?
+
+By default, it doesn't appear that ArcGIS Pro will give you a warning like ArcMap does. You can, however, enable a little warning popup in the top right of the window (where you've been ignoring that "A software update is available for ArcGIS Pro" message for weeks now):
+
+![ArcGIS Pro warning dialog]({% link images/transformations_prowarning.jpg %}){: .flex .flex--center}
+
+In the Pro Options dialog, go to the `Application` -> `Map and Scene` entry. Expand the `Spatial Reference` section; here you can choose your default spatial reference for new maps. Below this, there is a checkbox labeled `Warn if transformation between geographic coordinate system is required to align data sources correctly.`
+
+![Enabling warning in ArcGIS Pro]({% link images/transformations_prowarningsetting.jpg %}){: .flex .flex--center}
+
+## Show Me The Goods
+
+If you want to see this in action, load our [county boundaries layer](https://gis.utah.gov/data/boundaries/citycountystate/#CountyBoundaries) from both Open SGID (`opensgid.boundaries.county_boundaries`) and AGOL (`Utah County Boundaries`). Make sure the default `WGS 1984 (ITRF00) To NAD 1983` transformation is selected. Then run the `Split` tool between the two layers on the `NAME` field and look at one of the output feature classes. You'll see extra features for most or all of the surrounding counties with pretty small `shape_Area`'s:
```

Comment:
**chore**: internal links should use the [link syntax](https://github.com/agrc/gis.utah.gov/#link-to-post)

Reply:
Sometimes I forget those are internal links. I'll go through and update them 👍